db/helpers.go

```diff
@@ -202,15 +207,34 @@ func (ms *MongoStorage) createIndexes() error {
 	}); err != nil {
 		return fmt.Errorf("failed to create index on invitationCode for organization invites: %w", err)
 	}
+	// create an index for the orgAddress
+	orgParticipantdAddresIndex := mongo.IndexModel{
+		Keys: bson.D{
+			{Key: "orgAddress", Value: 1}, // 1 for ascending order
+		},
+	}
+	if _, err := ms.orgParticipants.Indexes().CreateOne(ctx, orgParticipantdAddresIndex); err != nil {
+		return fmt.Errorf("failed to create index on orgAddress and participantNo for orgParticipants: %w", err)
+	}
+	// create an index for the ParticipantNo
+	orgParticipantdNoIndex := mongo.IndexModel{
+		Keys: bson.D{
+			{Key: "participantNo", Value: 1}, // 1 for ascending order
+		},
+	}
+	if _, err := ms.orgParticipants.Indexes().CreateOne(ctx, orgParticipantdNoIndex); err != nil {
+		return fmt.Errorf("failed to create index on orgAddress and participantNo for orgParticipants: %w", err)
+	}
+
 	// create an index for the tuple OrgParticipant:ID and CensusID
-	orgParticipantNoIndex := mongo.IndexModel{
+	orgParticipantdAddresNoIndex := mongo.IndexModel{
 		Keys: bson.D{
 			{Key: "orgAddress", Value: 1},    // 1 for ascending order
 			{Key: "participantNo", Value: 1}, // 1 for ascending order
 		},
 		Options: options.Index().SetUnique(true),
 	}
-	if _, err := ms.orgParticipants.Indexes().CreateOne(ctx, orgParticipantNoIndex); err != nil {
+	if _, err := ms.orgParticipants.Indexes().CreateOne(ctx, orgParticipantdAddresNoIndex); err != nil {
 		return fmt.Errorf("failed to create index on orgAddress and participantNo for orgParticipants: %w", err)
 	}
```

Comment:
You can use `CreateMany` with a `[]mongo.IndexModel` instead of call `CreateOne` repeatedly.